.github/workflows/android-fabric-build.yml

```
@@ -0,0 +1,54 @@
+name: Build Android Fabric Example
+
+on:
+  pull_request:
+    branches:
+      - master
```

Let's change all triggers including iOS and Android to manual dispatch. We can run each of them according to the specific PR that needs it. This can also avoid CI costs.

```suggestion
on: workflow_dispatch
```